### ISSUE TITLE
NV41MZ fails to update via fwupd/LVFS

### ISSUE THREAD

**Issue by dennis1248** [Dasharo/dasharo-issues#68]
**Dasharo version**
v1.0.0

**Dasharo variant**
Clevo NV41MZ

**Affected component(s) or functionality**
Updates via fwupd/LVFS

**Brief summary**
Upon utilizing fwupd to search for firmware updates it will give me the following error;
```bash
╭─dennis@core ~ 
╰─$ sudo fwupdmgr get-updates
WARNING: UEFI capsule updates not available or enabled in firmware setup
  See https://github.com/fwupd/fwupd/wiki/PluginFlag:capsules-unsupported for more information.
Devices with no available firmware updates: 
 • SSD 980 PRO 500GB
No updatable devices
```
fwupd seems to be unable to update the firmware nor detect the UEFI as upgradable even though there are new versions available. I am running 1.0.0 and 1.0.1 is available via LVFS.

I am experiencing this issue under Fedora 35.

The error occurs both with Secureboot enabled and disabled.

**How reproducible**
Always, at least on my specific machine and with the current installation of Fedora 35.

**How to reproduce**

1. Run `fwupdmgr get-updates` via your shell under GNU/Linux
2. Wait for it to finish running
3. Notice the `UEFI capsule updates not available or enabled in firmware setup` warning and the lack of any acknowledgement of available UEFI firmware updates nor the device being listed as upgradable

**Expected behavior**
fwupd should successfully detect the UEFI as upgradable without any errors

**Actual behavior**
fwupd errors and does not list the UEFI as upgradable

**Additional context**
- @[wessel-novacustom](https://github.com/wessel-novacustom) also reported experiencing this issue under Ubuntu.
- Existing [fwupd issues](https://github.com/fwupd/fwupd/issues/3238) suggest this could be a kernel or UEFI issue. I can confirm that the default Fedora 35 kernel which I use is compiled with `CONFIG_EFI_ESRT=y`.

**Solutions you've tried**
- Disabling Secureboot

**Comment by dennis1248** [comment]
Should be fixed in https://github.com/Dasharo/fwupd/commit/1ad6f5156dae1d54963bd3999a89a86567216af2 and https://github.com/fwupd/fwupd/commit/d3ce827f967da30aa0fb35159f97f97be312729a over at upstream.

**Comment by dennis1248** [comment]
https://github.com/fwupd/fwupd/commit/ab06e034c35adb2c5e1c47f87161f04fe1ac7291 should have landed in the releases since the pull request was merged, yet I am having no luck with updating the firmware, the UEFI is still not detected as an upgradable device.

I tried with all fwupd releases since (1.7.5, 1.7.6, 1.7.7, 1.8.0) without any luck.

**Comment by dennis1248** [comment]
That mainline stable flashrom not supporting the Novacustom yet was indeed the problem, thank you.

It wasn't ideal for me to start hacking on my Fedora Silverblue installation so I instead grabbed a Ubuntu 20.04 installation media and installed the binaries you provided on it. After doing this it detected everything properly and I managed to do a successful update.
![Screenshot from 2022-05-05 19-47-02](https://user-images.githubusercontent.com/36002865/167015136-6337c151-80fa-4fdb-b5d8-684b2f7cdfd3.png)

Is it known when we can expect an updated flashrom release to be made available? I have a few more of these v1.0.0 Novacustoms "running in production". There is no urge to update them as we are not running in to any issues, it is just so I can keep an eye on these devices when to do start autoupdating their firmware.

I am closing the issue as the primary question has been answered and the problem has been solved.

**Comment by macpijan** [comment]
> https://github.com/fwupd/fwupd/commit/d3ce827f967da30aa0fb35159f97f97be312729a should have landed in the 1.8.0 release, yet I am having no luck with updating the firmware under Fedora Rawhide,

fwupd is one part of the equation. And it gets released quite frequently. The second one is to have proper flashrom version. And the problem with flashrom is that the last "official" release was more than two years ago: https://github.com/flashrom/flashrom/tree/v1.2  and this is what distributions pick up by default. This version, does not support Tiger Lake chipset.
